plugins/woocommerce-blocks/assets/js/data/payment/types.ts

```diff
@@ -19,10 +19,16 @@ import type { EventObserversType } from '../../base/context/event-emit';
 import type { DispatchFromMap } from '../mapped-types';
 import * as actions from './actions';
 
+export interface CreditCardNetworks {
```

Comment:
Currently, it is only for credit cards. But since it is possible for co-branded debit cards also to exist and be supported by Stripe in the future, I will rename this 👍